### ISSUE TITLE
Finish setting up fastpages for Kubeflow

### ISSUE THREAD

**Issue by jlewi** [kubeflow/fastpages#2]
Per kubeflow/community#316 we decided to setup fastpages to start experimenting it with as alternative to medium for blogging.

The repo has been created from the template but we still have to do all the remaining work to set this up.

**Comment by hamelsmu** [comment]
👋  I can help with setup.  If you wanto to give me admin access I can finish the setup process.  I just need to add SSH keys etc.  

@jlewi can you create the subdomain `blog.kubeflow.org` ?  That way I can add the CNAME file

**Comment by jlewi** [comment]
@hamelsmu if you need access to the repo: follow my instructions in comment
https://github.com/kubeflow/fastpages/pull/1#issuecomment-658463182

if you need admin privileges; ok but it would be better to use the least privileges necessary.

I can create the "blog.kubeflow.org" domain; but what do I point it at? e.g. am I pointing it an IP?

**Comment by jlewi** [comment]
Looks like I just follow
https://dev.to/trentyang/how-to-setup-google-domain-for-github-pages-1p58

To setup the subdomain.

**Comment by hamelsmu** [comment]
> Will we need Netlify for blog.kubeflow.org?

No, you only need Netlify if you want to have CI that will allow you view a live preview of your site.  This is completely optional, its just useful when you get PRs incase you don't want to clone and preview locally. 

> I can create the "blog.kubeflow.org" domain; but what do I point it at? e.g. am I pointing it an IP?

Yes you need to configure such that there is an A record, and you only need to point your apex domain at the following IPs

```
185.199.108.153
185.199.109.153
185.199.110.153
185.199.111.153
```

For GitHub Pages, here is an example with Google Domains, for example for the site docs.mlops-github.com I have the following setup

![image](https://user-images.githubusercontent.com/1483922/87487969-f49ee400-c5f3-11ea-86b8-c46f26853254.png)

**Comment by hamelsmu** [comment]
> if you need admin privileges; ok but it would be better to use the least privileges necessary.

You can give me admin privilege on just this repo temporarily and then revoke it after the merge, alternatively, you can add the SSH keys etc yourself and I don't need to do it.  Was just offering incase it is helpful

**Comment by hamelsmu** [comment]
> Can we do the following stages

> Get it setup on some GitHub generated URL
> Configure blog.kubeflow.org

Yes absolutely, you can do things in different stages, there is no problem
